apps/percona/deployers/percona-installer.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: localhost
+  connection: local
+
+  tasks:
+    - name: Deploying percona
+      shell: kubectl apply -f https://raw.githubusercontent.com/openebs/openebs/master/k8s/demo/percona/percona-openebs-deployment.yaml
```

Comment:
@shashank855 , Lets use a vars file to store things such as the URLs etc., They may change in future.